pkg/command/root.go

```diff
@@ -282,6 +261,56 @@ func ensureCLIInstanceID() (string, error) {
 	return cliID, nil
 }
 
+// isSkipCommand returns true if the command is part of the skip list by checking the prefix of
+// the command's command path matches with one of the item in the skip command list
+func isSkipCommand(skipCommandList []string, commandPath string) bool {
+	skipCommand := false
+	for _, cmdPath := range skipCommandList {
+		if strings.HasPrefix(commandPath, cmdPath) {
+			skipCommand = true
+			break
+		}
+	}
+	return skipCommand
+}
+
+// shouldSkipTelemetry checks if the command should be skipped for telemetry
+func shouldSkipTelemetry(cmd *cobra.Command) bool {
```

Comment:
How about renaming this to something like `shouldSkipTelemetryCollection`? As for the current function, the name is not clear whether this is about telemetry collection or telemetry sending.

Reply:
Makes sense. Updated with the function name to `shouldSkipTelemetryCollection`. Thanks!